bugwarrior/config.py

```diff
@@ -204,9 +205,26 @@ def get_taskrc_path(conf, main_section):
     )
 
 
-def get_data_path():
-    return os.path.expanduser(os.getenv('TASKDATA', '~/.task'))
+@lru_cache()
+def get_data_path(config, main_section):
+    taskrc = get_taskrc_path(config, main_section)
 
+    # We cannot use the taskw module here because it doesn't really support
+    # the `_` subcommands properly (`rc:` can't be used for them).
```

Comment:
Wouldn't it be better to send a patch to taskw? There seems to be code there to parse the taskwarrior config file, so it should be easy to have it parse the output of `task _show` instead of a file.